cognite/transformations_cli/commands/utils.py

```diff
@@ -67,7 +67,7 @@ def print_transformations(transformation: List[Transformation]) -> str:
                 t.name,
                 t.id,
                 t.external_id,
-                t.destination.type,
+                "alpha_data_model_instances" if t.destination.type == "data_model_instances" else t.destination.type,
```

Comment:
Not sure this kind of backwards compatibility thing (which I think it is?) is worth doing. I'm thinking probably not, tbh?

Reply:
It is actually not for backward compatibility. 
This function prints a list of transformations in tabular form. This change is for exposing `transformation.destination.type` with `alpha` prefix if it is `data_model_instances`  (in order to make it consistent with deployment manifest change in this PR: `data_model_instances` -> `alpha_data_model_instances`). Because jetfire-backend does not have this prefix currently, I change it here before printing. 

 I can remove this if it does not make sense.

Reply:
I think we will be moving to the new FDM API in the coming days. I think we should be careful about using `alpha_` prefix for the new destination types in `jetfire-backend`, as you said earlier.

Reply:
I see. It makes sense, but if it's just for making things easier/better for existing users of this alpha feature, I'd prefer simpler code at the cost of some potential pain for such users.

Reply:
It is just cosmetics, I'll remove it then 👍